broker/src/main/java/io/camunda/zeebe/broker/system/partitions/impl/steps/SnapshotDirectorPartitionTransitionStep.java

```diff
@@ -67,7 +68,8 @@ public ActorFuture<Void> transitionTo(
                 snapshotPeriod);
       }
 
-      final var future = context.getActorSchedulingService().submitActor(director);
+      final var future =
+          context.getActorSchedulingService().submitActor(director, SchedulingHints.ioBound());
```

Comment:
:wrench: I guess we learned from our recent discussion that the director doesn't need to be an IO-bound actor, but the SnapshotStore. I would suggest not to mark here the director as io bound.

Reply:
Yes, that's a good point! :+1:

Reply:
@Zelldon I'm thinking we should do the same for the `LogDeletionService` which is actually blocking longer than the snapshot store. Are you okay with just making the change in this pr?

Reply:
Doit!

Reply:
But I guess we need to verify that with an benchmark whether it makes an impact ? :thinking: